### ISSUE TITLE
After backup problems with settings 

### ISSUE THREAD

**Issue by EddyMI3d** [Birkbjo/OctoPrint-Themeify#45]
Hello there!

After I made a backup, in Themeify the settings do not appear, although these settings are still in the config.yaml and are also used to make the changes to the web GUI.
I also can not enter new settings to Themeify.

**Comment by Birkbjo** [comment]
I haven't been very active with octoprint lately, and haven't tested the new backup feature - but I will try to find time for this soon soon™

**Comment by EddyMI3d** [comment]
No problem.
Due to issues of other plugins, I'd installed all anew.
I assume a plugin messed up before I made the backup file

**Comment by cmarabate** [comment]
I didn't make a backup but I think I have the same issue. All the custom styles and custom tab-icons that I setup under "Advanced options" are gone and I am unable to add new ones. But everything that I saved before it got messed up are still active. I'm not really sure what caused it because all I was doing was adding a couple more Custom Tab-icons. The first one I added worked but the other one did not and it has been messed up ever since. I have rebooted the system, restarted OctoPrint, & Disabled and Re-enabled Themeify but none of that fixed the issue.

![image](https://user-images.githubusercontent.com/12236062/65821885-b45baa00-e209-11e9-9d35-5abe7d464d18.png)

As you can see from the screenshot, it also shows all the options under "General" as unchecked. But I have tried checking them, pressing save, and restarting OctoPrint and when I open the settings, they are unchecked again.

I'm out of ideas at this point, so hopefully you can help me resolve this issue without having to start from scratch. Otherwise, I am stuck with my current styling and tab-icons.

Thanks,

Chris

**Comment by cmarabate** [comment]
I fixed it!!! Thanks to @EddyMI3d mentioning that...

> these settings are still in the config.yaml

At first I had no clue how to find this file but after a ton of research I was eventually able to find it in **/home/pi/.octoprint/** and open it in a text editor. From there it was easy to identify the issue from looking at the code it added for each of my custom styles. I saw that the last rule I added had the domID set to a value that included double quotes inside it because instead of adding just the ID(#tab_plugin_octolapse_link) I wanted to style, I must have added the following by mistake...

> #id="tab_plugin_octolapse_link"

![WhatBrokeThemeify](https://user-images.githubusercontent.com/12236062/66346293-dcd95780-e91f-11e9-81ad-295aca7b5c7d.jpg)

This was apparently the cause of my entire issue because after editing config.yaml accordingly and rebooting the system, everything was fixed!
